near-sdk/src/json_types/public_key.rs

```diff
@@ -191,7 +203,7 @@ mod tests {
     use std::convert::TryInto;
 
     fn binary_key() -> Vec<u8> {
-        let mut binary_key = vec![0];
+        let mut binary_key = vec![CurveType::ED25519 as u8];
         binary_key.extend(
             bs58::decode("6E8sCci9badyRkXb3JoRpBj5p8C6Tw41ELDZoiihKEtp").into_vec().unwrap(),
```

Comment:
so since these tests are essentially checking themselves, this probably could just return the actual `Base58PublicKey` to make the tests cleaner. More of an opinion though, but with this change, the tests read a bit weird.